setup.py

```diff
@@ -12,14 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+from distutils.sysconfig import get_config_var, get_python_inc
 from setuptools import setup, Extension
 
 module = Extension(
     'ptxcompiler._ptxcompilerlib',
     sources=['ptxcompiler/_ptxcompilerlib.cpp'],
-    include_dirs=['/usr/local/cuda-11.5/include'],
+    include_dirs=[os.path.dirname(get_python_inc())],
     libraries=['nvptxcompiler_static'],
-    library_dirs=['/usr/local/cuda-11.5/lib64'],
+    library_dirs = [get_config_var("LIBDIR")],
```

Comment:
ditto

Reply:
This is actually where the libraries with the exception of those in the sysroot. We symlink [`libcuda.so`]( https://github.com/conda-forge/nvcc-feedstock/blob/295b65a0c67be9d81e7f4812cd495a9e95dcae57/recipe/install_nvcc.sh#L105 ) there. We then symlink the remaining one in the recipe

Reply:
Sounds good, thanks for reminders John!